cmake/thirdparty/get_cuml.cmake

```diff
@@ -37,7 +56,8 @@ endfunction()
 # Change pinned tag here to test a commit in CI
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
-find_and_configure_cuml(VERSION    21.12
-                        FORK       rapidsai
-                        PINNED_TAG branch-21.12
+find_and_configure_cuml(VERSION    ${RAPIDS_TRITON_MIN_VERSION_rapids_projects}
+                        FORK       dantegd
+                        PINNED_TAG 2112-libcuml-configurable
```

Comment:
This will be changed back to `branch-22.04` once https://github.com/rapidsai/cuml/pull/4296 is merged?

Reply:
We might want to pin the actual commit hash for that merge.

Reply:
More precisely, we are planning to create a particular branch to pin RMM, RAFT, cuML and RAPIDS-Triton to be entirely fixed for release 22.03 tomorrow or Friday